config/config.go

```diff
@@ -154,6 +155,9 @@ func (c *Config) Load(file string) error {
 			return err
 		}
 		c.FileConfig.AddConfigPath(home)
+		c.FileConfig.AddConfigPath(".")
+		c.FileConfig.AddConfigPath(xdg.ConfigHome)
+		c.FileConfig.AddConfigPath(xdg.ConfigHome + "/gnmic")
```

Comment:
@sulrich shouldn't `$PWD` be the first path to be searched for a config file?
meaning:
`c.FileConfig.AddConfigPath(".")`
`c.FileConfig.AddConfigPath(home)`
`c.FileConfig.AddConfigPath(xdg.ConfigHome)`
`c.FileConfig.AddConfigPath(xdg.ConfigHome + "/gnmic")`

Reply:
ahh good point.  i'll update.

Reply:
updated as you noted.

Reply:
thanks, appreciate the quick update!